src/Container/CommandBusFactory.php

```diff
@@ -22,21 +23,20 @@
  */
 class CommandBusFactory extends AbstractBusFactory
 {
-
     /**
      * @inheritdoc
      */
-    protected function getBusClass()
+    public function containerId()
     {
-        return CommandBus::class;
+        return 'command_bus';
     }
 
     /**
      * @inheritdoc
      */
-    protected function getBusConfigKey()
+    protected function getBusClass()
```

Comment:
also in this case the name is correct: https://github.com/prooph/service-bus/blob/master/src/Container/AbstractBusFactory.php#L61

The factory needs to know which bus class should be used to create the bus